.idea/compiler.xml

```diff
@@ -11,14 +11,6 @@
       <entry name="!?*.kt" />
       <entry name="!?*.clj" />
     </wildcardResourcePatterns>
-    <annotationProcessing>
-      <profile name="Maven default annotation processors profile" enabled="true">
-        <sourceOutputDir name="target/generated-sources/annotations" />
-        <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
-        <outputRelativeToContentRoot value="true" />
-        <module name="dokka-maven-plugin" />
-      </profile>
-    </annotationProcessing>
```

Comment:
Should this file be removed from source control ?

Reply:
Probably, but the decision was to keep the entire `.idea` directory to help setup this project, so we sometimes just update it

Reply:
I usually only commit the codestyle settings (`.idea/codeStyles/`). The problem of adding other files is that it creates noise in the diffs.